__tests__/app.test.js

```diff
@@ -0,0 +1,30 @@
+const db = require("../db/connection.js");
+const { app } = require("../app.js");
+const request = require("supertest");
+const seed = require("../db/seeds/seed");
+const testData = require("../db/data/test-data/");
+const { response } = require("express");
+
+afterAll(() => {
+  db.end();
```

Comment:
consider the asynchronous nature of this method.

Reply:
we need to return